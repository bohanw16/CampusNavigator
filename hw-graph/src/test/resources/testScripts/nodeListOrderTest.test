# nodeListOrder testing file
# Tests that a graph can be created,
# three nodes can be added to the graph,
# and a list of nodes can be printed out in correct order

# Create a graph
CreateGraph g1

# Add three nodes
AddNode g1 n1
AddNode g1 n2
AddNode g1 n3

# Print the nodes in the graph
ListNodes g1