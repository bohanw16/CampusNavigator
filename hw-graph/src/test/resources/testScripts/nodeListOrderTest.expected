# nodeListOrder testing file
# Tests that a graph can be created,
# three nodes can be added to the graph,
# and a list of nodes can be printed out in correct order

# Create a graph
created graph g1

# Add three nodes
added node n1 to g1
added node n2 to g1
added node n3 to g1

# Print the nodes in the graph
g1 contains: n1 n2 n3